testsuite/tests/typing-modes/crossing.ml

```diff
@@ -129,7 +129,7 @@ let cross_local (x : cross_local @ local) : _ @ global = x
 Line 1, characters 57-58:
 1 | let cross_local (x : cross_local @ local) : _ @ global = x
                                                              ^
-Error: This value escapes its region.
+Error: This value is local to the parent region but expected to be "global".
```

Comment:
@goldfirere suggests that we could print both meesages (specifically for locality)

Reply:
Instead of "parent region", use the hint system to specify either "caller region" or "outer region". (future work)